apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.IntegrationTests/BigtableFixture.cs

```diff
@@ -32,7 +32,9 @@ public class BigtableFixture : IDisposable, ICollectionFixture<BigtableFixture>
         private const string TestInstanceEnvironmentVariable = "BIGTABLE_TEST_INSTANCE";
         private const string TestProjectEnvironmentVariable = CloudProjectFixtureBase.TestProjectEnvironmentVariable;
 
-        public const string ColumnFamily1 = "cf1";
+        public const string DefaultColumnFamily = "cf1";
+        public const string DefaultQualifier = "row_exists";
```

Comment:
I'm still somewhat confused about what a qualifier is, but I think that's Bigtable-ness rather than anything in this API... is "row_exists" a useful constant outside tests? Is it something we should put in the public API anywhere?

Reply:
There's no significance here. It could have just been called "foo". I figured it might be good to have some meaning for the default case (kind of like how in one of the the `CheckAndMutateRow` tests, I use a column name of "original_row_status" and then values or "row_had_cells" and "row_had_no_cells"). If you think there's too much opportunity for confusion though, I'm happy to change them to more nonsensical things.

Reply:
I think it's fine - I'll just read up on what qualifiers are in the context of Bigtable.

Reply:
Basically, as I understand it, column families are what you would think of as columns in a relational DB table: they are fixed and relatively expensive to change. Within families, you have cells with arbitrary qualifier/value byte strings, like key/value pairs (they are really key/value/version tuples though because you can update the value associated with a qualifier and the history of old values is maintained). This means qualifiers can contain data as well as the values, like in [this example](https://cloud.google.com/bigtable/docs/overview#storage-model) from the Bigtable docs.

Reply:
Aha. I'd just think of that as a column name, scoped to the column family. Right - good to know what's going on, thanks.

Reply:
Yes, sometimes they're called 'column qualifiers' and other times they're just called 'qualifiers'. I've kind of settled on the latter, but I'll rename this to `DefaultColumnQualifier` so its clear.